packages/get-starknet/src/constants.ts

```diff
@@ -1,10 +1,16 @@
 
 // The supported RPC version should be update base on the Provider.
 // With Provider `Alchemy`, it is 0.7
 export const SupportedStarknetSpecVersion = ['0.7'];
+
+// The wallet API support is 0.7.2 but the RPC specs requests xx.yy. Hence we skip the last digits.
```

Comment:
i believe WalletApi is different with RPC
it is talking about the wrapper API version (wallet api versions)

Reply:
we can use [0.7] for now , but await for clarify